.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "anserini-eval"]
+	path = anserini-eval
```

Comment:
path should be `eval/` for consistency across repos

Reply:
I have updated it.